### ISSUE TITLE
Intuitions for padding model output columns?

### ISSUE THREAD

**Issue by ghost** [havakv/pycox#152]
Thanks for creating this wonderful package!

I have a few questions regrading the functions for deephit model:
https://github.com/havakv/pycox/blob/master/pycox/models/deephit.py

**1. Regarding `predit_surv` in line 155**
`surv = 1. - cif.sum(0)`
why it needs to take the sum for `cif`? The cumulative sum has already been calculated in cif, and in my view, it should be:
`surv = 1 - cif`

**2. Regarding `predict_pmf` in line 202**
`pmf = pad_col(preds.view(preds.size(0), -1)).softmax(1)[:, :-1]`
what are the intuitions for padding another column for the model output before softmax? 

In the original paper "Continuous and Discrete-Time Survival Prediction with Neural Networks", page 7, it says 
`Alternatively, one could let φm+1(x) vary freely, something that is quite common in machine learning, but we chose to follow the typical conventions in statistics.`

I am confused about this trick. If I took the  φm+1(x) very large, then the estimated probability (after applying softmax) for each time interval would be very small, will this affect my predictions?

Any comments would be appreciated! Thanks in advance!
